src/toil/wdl/wdltoil.py

```diff
@@ -5180,8 +5401,8 @@ class WDLStartJob(WDLSectionJob):
 
     def __init__(
         self,
-        target: WDL.Tree.Workflow | WDL.Tree.Task,
-        inputs: WDLBindings,
+        target: Union[WDL.Tree.Workflow, WDL.Tree.Task],
```

Comment:
I think using `|` everywhere instead of `Union` is another pyupgrade change that we want to keep.